libs/TaskManager.js

```diff
@@ -149,6 +153,19 @@ module.exports = class TaskManager{
 			if (task){
 				this.addToRunningQueue(task);
 				task.start(() => {
+					if(task.webhook && task.webhook.length > 3){
```

Comment:
Old habit.. seeing if it's actually any thing there could prolly add a URL regex